internal/datastore/postgres/readwrite.go

```diff
@@ -53,69 +57,220 @@ type pgReadWriteTXN struct {
 	newXID xid8
 }
 
+func appendForInsertion(builder sq.InsertBuilder, tpl *core.RelationTuple) sq.InsertBuilder {
+	var caveatName string
+	var caveatContext map[string]any
+	if tpl.Caveat != nil {
+		caveatName = tpl.Caveat.CaveatName
+		caveatContext = tpl.Caveat.Context.AsMap()
+	}
+
+	valuesToWrite := []interface{}{
```

Comment:
```go
[]any{
    ...
}
```

Reply:
My linter complains about this, but obviously the one in CI doesn't.